### ISSUE TITLE
Preview shows wrong numbers for BigInts

### ISSUE THREAD

**Issue by stefanbrugger** [johan/QuickJSON#6]

**Comment by stefanbrugger** [comment]
<img width="1097" alt="Screen Shot 2021-11-19 at 11 55 59" src="https://user-images.githubusercontent.com/5682925/142611439-e03c6564-8ddf-40ed-9c23-9a16a5821df5.png">


It seems that the java script rendering overflows for BigInts showing incorrect numbers in the preview

**Comment by johan** [comment]
Yeah; it's cheating by using the language itself to parse and re-serialize the JSON – so it drops input to `Number` resolution, rather than tokenizing and reformatting the raw data as strings.
